Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14
```

Comment:
The comment 'FROM node:16' should be placed above the line of code 'FROM node:16' for better readability.